commons/addons/elastic.md

```diff
@@ -57,7 +57,15 @@ Learn more on [APM official documentation](https://www.elastic.co/guide/en/apm/g
 
 ### How to setup APM
 
-Any applications linked to the APM application will have the right credentials and APM endpoint automatically avaiable as environment variables. These variables can be picked up automatically by the APM agent you are using in your application. As such everything should work automatically.
+Any applications linked to the APM application will have the right credentials and APM endpoint automatically available as environment variables. These variables can be picked up automatically by the APM agent you are using in your application. As such everything should work automatically except for Java that needs a bit more configuration.
```

Comment:
> except for Java that needs a bit more configuration.
It does not have to be like this. You can actually add this to your dependencies and it will work.

The auto agent config is just another agnostic way of doing it.

Reply:
Alright, I removed that part